Toontown_Tower_Defense.pde

```diff
@@ -178,16 +178,38 @@ String gag()
     return "Drop";
 }
 
+//  Work on shortening this
 void mouseClicked()
 {
   gagSelected = gagClicked();
   if (gagSelected)
     gag = gag();
   else
   {
+    for (int i = 0; i < 42; i++)
+    {
+      if (!towers[i] && mouseX >= towersX[i] && mouseX <= towersX[i]+80
+                     && mouseY >= towersY[i] && mouseY <= towersY[i]+80)
+      {
+        towers[i] = true;
+        towersGags[i] = gag;
+        break;
+      }
+    }
+    
+    /*
```

Comment:
No need to hide in a comment. Just delete everything. (The past history is preserved on GitHub / Git.)